src/style.css

```diff
@@ -0,0 +1,28 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  margin: 2px 6px 2px 2px;
+  width: 12px;
+  height: 12px;
+  background-image: url(images/star.svg);
+  background-position: cover;
+  display: inline-block;
```

Comment:
if the parent has `display: flex` then display for direct children will not be applied, so this is redundant
JFYI